wrapper/src/main/java/software/amazon/jdbc/PropertyDefinition.java

```diff
@@ -95,7 +95,20 @@ public static Collection<AwsWrapperProperty> allProperties() {
     return PROPS_BY_NAME.values();
   }
 
-  public static void removeAll(Properties props) {
```

Comment:
I'd keep this method and create a new one removeAllExceptCredentials()

Reply:
re-added!